### PR TITLE
[snmp] extend snmp test for xcvrd

### PR DESCRIPTION
**Pull request by keboliu**
### Description of PR

Summary:
Extend snmp test case to support xcvrd snmp test

### Type of change

- [] Bug fix
- [] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### How did you do it?
Add new test cases in snmp test to test xcvrd feature
detail see test plan [https://github.com/Azure/SONiC/blob/gh-pages/doc/xcvrd-snmp-testbed-test-plan.md](url)

#### How did you verify/test it?
run test on mlnx platform.

### Documentation 
Test plan link:  [https://github.com/Azure/SONiC/blob/gh-pages/doc/xcvrd-snmp-testbed-test-plan.md](url)

### CODE REVIEW

File: ansible/roles/test/tasks/snmp/phys_table.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+- name: Check chassis entity exists in snmp
+  fail: msg="No chassis entry in Physical Table MIB"
+  when: "{{ chassis_id not in snmp_physical_entities or snmp_physical_entities[chassis_id].entPhysClass != 3 }}"
+
+- debug:
+    msg: "{{ snmp_physical_entities.keys()  }}"
+
+- name: Check all transceivers present in Entity MIB
+  fail:
+    msg: '{{ (((item.value.name.split("Ethernet")[-1] | int + 1) * 1000) | string) }}'
```
Comment (qiluo-msft) [inline ansible/roles/test/tasks/snmp/phys_table.yml]: Could you explain the complex expression?

Comment (keboliu) [reply ansible/roles/test/tasks/snmp/phys_table.yml]: @qiluo-msft the OID index is defined as ifindex*1000, and for EthernetN, ifindex is (N+1), so the OID index for EthernetN will be (N +1)*1000,  detail info can be found in the requirement https://github.com/Azure/SONiC/blob/gh-pages/doc/OIDsforSensorandTransciver.MD#transceiver-requirements-entity-mib

Comment (qiluo-msft) [reply ansible/roles/test/tasks/snmp/phys_table.yml]: The comment is perfect for code comment. Every future code reader will have the same problem

Comment (keboliu) [reply ansible/roles/test/tasks/snmp/phys_table.yml]: @qiluo-msft will add this to code as comments.

**Comment by qiluo-msft** [comment]
It will break the nightly test of old images. Could you add some logic to bypass the test?

**Pull request merged**